### ISSUE TITLE
Local Hotseat Multi: Pick a spell bug

### ISSUE THREAD

**Issue by jdoleary** [jdoleary/Spellmasons#53]
>  when one of us was dead, every turn it asked the dead player to pick a spell. This spell then went to the alive player. Is this intended or a bug of some sort? Local hotseat.

Reported by SolemnSundayBand on Reddit

**Comment by jdoleary** [comment]
This may have been resolved in 1.25.  Todo:Verify

**Comment by jdoleary** [comment]
Tested with 3 hotseat players.  1 and 2 are dead.  It still switches to 1's turn but you can't do anything but end turn, then on 2's turn it offers them an upgrade and errors:
```
Underworld.ts:2387 Unexpected: Cannot choose upgrades, upgrades left to choose: 0
endMyTurnButtonHandler @ Underworld.ts:2387
handleInputDown @ eventListeners.ts:219
keydownListener @ eventListeners.ts:116
Underworld.ts:2390 Unexpected: Attempted to show upgrades but cannot
```
but the original bug as stated has not occurred.
